packages/sanity/src/presentation/constants.ts

```diff
@@ -10,26 +11,13 @@ export const EDIT_INTENT_MODE = 'presentation'
 export const MAX_TIME_TO_OVERLAYS_CONNECTION = 3_000 // ms
 
 // The API version to use when using `@sanity/client`
-export const API_VERSION = '2023-10-16'
+export const API_VERSION = apiVersion
```

Comment:
Aligning on the `apiVersion` used here increases the potential effect of https://github.com/sanity-io/client/pull/990